avocado/core/tree.py

```diff
@@ -445,7 +445,7 @@ def process_node(node):
         Generate this node's tree-view
         :return: list of lines
         """
-        if node.multiplex:
+        if getattr(root, "multiplex", False):
```

Comment:
I planned to remove single `ascii` method and create TreeNode method instead, but it's not yet part of this PR.